### PR TITLE
More readme changes

### PR DESCRIPTION
**Pull request by salmart-dev**
Address some of the recurring issues + make task text a bit funny?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,25 @@
 # Trial task for back-end developers
 
-Please use Node.js 12+, MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. It is recommended to use some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
+Someone from support notified us that users are using too many times passwords like "password", "dog123", or "heeello" and their accounts are getting hacked by their neighbours to see private pictures of their private parts.
+We need this fixed ASAP, but our developers are busy working on the next-cool-feature™ using the next-cool-stack™, and they have no time for implementing the changes needed to fix the issue. So our only chance is YOU!*
+
+Your objective for this task is to build two components:
+a REST API that receives a password and gives information about its validity, given a set of rules
```
Comment (jleeothon) [inline README.md:7]: Missing bullets (- or * or +)

File: README.md [hunk 2]
```diff
@@ -1,18 +1,25 @@
 # Trial task for back-end developers
 
-Please use Node.js 12+, MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. It is recommended to use some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
+Someone from support notified us that users are using too many times passwords like "password", "dog123", or "heeello" and their accounts are getting hacked by their neighbours to see private pictures of their private parts.
+We need this fixed ASAP, but our developers are busy working on the next-cool-feature™ using the next-cool-stack™, and they have no time for implementing the changes needed to fix the issue. So our only chance is YOU!*
```
Comment (jleeothon) [inline README.md:4]: Note that in most flavours of Markdown, a newline does not mean a new paragraph. (You need two newlines to mark a new paragraph).

Don't know if you mean this as a new paragraph or not.

File: README.md [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
+a REST API that receives a password and gives information about its validity, given a set of rules
+a script that reads the already existing passwords from the database and sets a validity flag based on the API you implement
+
+We ask you to use Node.js 12+ (TypeScript is an option), MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. We recommend using some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
 
 What we value when reviewing your task:
 - attention to perfomance and scalability
 - a good README with instructions on how to run your execution
 - good code architecture
 - polished result: not too many typos or commented code and consistent styling
```
Comment (jleeothon) [inline README.md:16]: Nitpick to avoid ambiguity: consider "consistent styling, and not too many typos or commented code"

File: README.md [hunk 4]
```diff
@@ -25,17 +32,17 @@ The ruleset the passwords have to be validated against is:
...
-The script must set the `valid` field to 1 or 0 for valid and invalid passwords.
+The script must also set the `valid` field to 1 or 0 for valid and invalid passwords in the given MySQL table.
 
 Please provide and document a possibility to change the MySQL connection settings used by the script.
 
-## 3 (BONUS) - Checking for compromised passwords
+## Task 3 (BONUS) - Checking for compromised passwords
 
-Extend your script to check each password not only for validity but also if it has been compromised in the past. Please use the provided compromised password API for this.
+Extend the script you produced for Task 2 to include a check that uses a second API which tells you if passwords have been compromised in the past. This second API is already provided (see the next paragraph).
```
Comment (jleeothon) [inline README.md:45]: Nitpick: if we need to suggest "see the next paragraph", maybe there's some else to improve in how this is explained.

We probably don't need a level-3 title "Using the compromised password API"

Consider something more or less like:

> (...) have been compromised in the past. 
>
> This second API is already implemented for you in this Docker image (...)

File: README.md (hunk 1 above)
Comment (klarkent) [reply README.md:7]: Ohhh Grammarly must've stripped something away when I copied, I guess.

File: README.md (hunk 2 above)
Comment (klarkent) [reply README.md:4]: No, it's not meant to be a new paragraph as it's a continuation of the topic before. I broke it into a new line for ease of editing.

File: README.md (hunk 4 above)
Comment (klarkent) [reply README.md]: Ok fixed

File: README.md (hunk 3 above)
Comment (klarkent) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (klarkent) [reply README.md]: fixed

**Pull request merged**